source/features/conflict-marker.css

```diff
@@ -2,3 +2,7 @@
 	vertical-align: middle;
 	color: #c0c0c0;
 }
+
+[data-color-mode='dark'] .rgh-conflict-marker svg {
+	color: #505050;
```

Comment:
This could use a variable too, the color doesn't need to be exactly what it was. If it's near enough as long as it looks good